<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerProf</class>
 <widget class="QMainWindow" name="customerProf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 229, 199);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>951</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton {
font: 20pt &quot;MS Shell Dlg 2&quot;;
padding-left: 20px;
padding-right: 20px;
border-radius: 5px;
	background-color: rgb(180, 180, 180);
color: blue;
}

</string>
       </property>
       <property name="text">
        <string>خروج</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton {
font: 20pt &quot;MS Shell Dlg 2&quot;;
padding-left: 20px;
padding-right: 20px;
border-radius: 5px;
	background-color: rgb(180, 180, 180);
color: blue;
}

</string>
       </property>
       <property name="text">
        <string>سوابق فروش</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_2 {
	font: 20pt &quot;MS Shell Dlg 2&quot;;
padding-left: 20px;
padding-right: 20px;
border-radius: 5px;
background-color: rgb(180, 180, 180);
color: blue;
}

</string>
       </property>
       <property name="text">
        <string>لیست محصولات</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_3 {
	font: 20pt &quot;MS Shell Dlg 2&quot;;
padding-left: 20px;
padding-right: 20px;
border-radius: 5px;
background-color: rgb(180, 180, 180);
color: blue;
}

QPushButton#pushButton_3:hover {
	background-color: rgb(43, 255, 185);
border-style: solid;
 border-width: 1px;
	border-color: rgb(245, 179, 1);
	color: rgb(241, 182, 88);
}
</string>
       </property>
       <property name="text">
        <string>طرح های تبلیغاتی</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(53, 66, 89);</string>
    </property>
    <property name="title">
     <string> پروفایل</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>100</y>
       <width>111</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>نام فروشگاه:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>170</y>
       <width>101</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>شماره تلفن:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>230</y>
       <width>61</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>آدرس:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>51</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>نقش:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>91</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>نام کاربری:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>71</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>پسورد:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_4 {
	font: 20pt &quot;MS Shell Dlg 2&quot;;
padding-left: 20px;
padding-right: 20px;
border-radius: 5px;
background: yellow;
color: blue;
}

QPushButton#pushButton_4:hover {
	background-color: rgb(43, 255, 185);
border-style: solid;
 border-width: 1px;
	border-color: rgb(245, 179, 1);
	color: rgb(241, 182, 88);
}
</string>
     </property>
     <property name="text">
      <string>ذخیره تغییرات</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>91</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>231</y>
       <width>256</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Lalezar&quot;;
background-color: rgb(116, 141, 166);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Lalezar&quot;;</string>
     </property>
     <property name="text">
      <string>******</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
